public/app/features/canvas/elements/server.tsx

```diff
@@ -0,0 +1,474 @@
+              />
+              <g transform="matrix(.50833 0 0 1 27.315 0)">
+                <rect x="55.556" y="57.472" width="107.76" height="5.7472" />
+                <rect x="55.558" y="86.141" width="107.76" height="5.7472" />
+                <g className={styles.thinLine}>
+                  <rect x="74.357" y="73.261" width="96.957" height="2.8736" />
+                  <rect x="74.379" y="101.9" width="96.957" height="2.8736" />
+                </g>
+              </g>
+              <circle
+                className={styles.circle}
+                cx="83.858"
+                cy="178.2"
+                r="5.9401"
+                style={{ animation: bulbAnimation, fill: bulbColor }}
+              />
+            </g>
+            <g transform="translate(-1.9978 -7.5028)">
+              <path className={styles.monitor} d="m103.22 75.305h118.87v76.914h-118.87z" />
+              <path
+                className={styles.monitorOutline}
+                d="m103.22 70.305-5 5v76.916l5 5h118.87l5-5v-76.916l-5-5zm5 10h108.87v58.916h-108.87z"
+              />
+            </g>
+            <path d="m135.6 148.14h50.113l4 15.156h-58.113z" />
+            <path className={styles.keyboard} d="m118.23 183.19h88.848l24 19.476h-136.85z" />
+          </g>
+        </>
+      ) : null}
+    </svg>
+  );
+};
+
+export const serverItem: CanvasElementItem<ServerConfig, ServerData> = {
+  id: 'server',
+  name: 'Server',
+  description: 'Basic server with status',
+
+  display: ServerDisplay,
+
+  defaultSize: {
+    width: 100,
+    height: 155,
+  },
+
+  getNewOptions: (options) => ({
+    ...options,
+    background: {
+      color: {
+        fixed: 'transparent',
+      },
+    },
+    placement: {
+      width: options?.placement?.width ?? 100,
+      height: options?.placement?.height ?? 155,
+      top: options?.placement?.top,
+      left: options?.placement?.left,
+    },
+  }),
+
+  // Called when data changes
+  prepareData: (ctx: DimensionContext, cfg: ServerConfig) => {
+    const data: ServerData = {
+      blinkRate: cfg?.blinkRate ? ctx.getScalar(cfg.blinkRate).value() : 0,
+      statusColor: cfg?.statusColor ? ctx.getColor(cfg.statusColor).value() : 'transparent',
+      bulbColor: cfg?.bulbColor ? ctx.getColor(cfg.bulbColor).value() : 'green',
+      type: cfg?.type ?? ServerTypes.Single,
+    };
+
+    return data;
+  },
+
+  registerOptionsUI: (builder) => {
+    const category = ['Server'];
+    builder
+      .addRadio({
+        category,
+        path: 'config.type',
+        name: 'Type',
+        settings: {
+          options: [
+            { value: ServerTypes.Single, label: 'Single' },
+            { value: ServerTypes.Stack, label: 'Stack' },
+            { value: ServerTypes.Database, label: 'Database' },
+            { value: ServerTypes.Terminal, label: 'Terminal' },
+          ],
+        },
+        defaultValue: ServerTypes.Single,
```

Comment:
This default value doesn't seem to be initialized correctly for me 🤔 (no option is highlighted initially, I have to manually click the single option for it to be highlighted)
![Screenshot 2023-01-11 at 1 41 57 PM](https://user-images.githubusercontent.com/22381771/211923994-c6bcf30c-810a-402f-9ea4-3eb795e1312a.png)
![Screenshot 2023-01-11 at 1 42 02 PM](https://user-images.githubusercontent.com/22381771/211923998-582a1909-8560-44cf-8473-b93e1a241bef.png)